conf/streamline.yaml

```diff
@@ -9,7 +9,7 @@ modules:
     className: com.hortonworks.streamline.streams.service.StreamsModule
     config:
       #change the below to the path on your local machine
-      streamlineStormJar: /tmp/streamline-runtime-storm-0.1.0-SNAPSHOT.jar
+      streamlineStormJar: ${STREAMLINE_HOME}/libs/streamline-runtime-storm-0.1.0-SNAPSHOT.jar
```

Comment:
@HeartSaVioR we could make the config nullable and keep the value in streamline-dev.yaml and take it out of streamliine.yaml and use streamline-dev.yaml for debugging purpose